packages/flutter/lib/src/painting/strut_style.dart

```diff
@@ -340,27 +340,28 @@ class StrutStyle with Diagnosticable {
     double? fontSize,
     double? height,
     TextLeadingDistribution? leadingDistribution,
-    this.leading, // TextStyle does not have an equivalent (yet).
+    double? leading, // TextStyle does not have an equivalent (yet).
     FontWeight? fontWeight,
     FontStyle? fontStyle,
-    this.forceStrutHeight,
+    bool? forceStrutHeight,
     String? debugLabel,
     String? package,
-  }) : assert(fontSize == null || fontSize > 0),
-       assert(leading == null || leading >= 0),
-       assert(package == null || fontFamily != null || fontFamilyFallback != null),
-       fontFamily = fontFamily != null
-           ? (package == null ? fontFamily : 'packages/$package/$fontFamily')
-           : textStyle.fontFamily,
-       _fontFamilyFallback = fontFamilyFallback ?? textStyle.fontFamilyFallback,
-       height = height ?? textStyle.height,
-       leadingDistribution = leadingDistribution ?? textStyle.leadingDistribution,
-       fontSize = fontSize ?? textStyle.fontSize,
-       fontWeight = fontWeight ?? textStyle.fontWeight,
-       fontStyle = fontStyle ?? textStyle.fontStyle,
-       debugLabel = debugLabel ?? textStyle.debugLabel,
-       _package = package; // the textStyle._package data is embedded in the
-  // fontFamily names, so we no longer need it.
+  }) : this(
+         fontFamily: fontFamily != null
+             ? (package == null ? fontFamily : 'packages/$package/$fontFamily')
+             : textStyle.fontFamily,
```

Comment:
I guess this is not a bug introduced in this patch, but the fontFamilyFallback needs to embed the package path too.

Reply:
I think it does end up doing this but through a `getter`.

```
List<String>? get fontFamilyFallback {
  if (_package != null && _fontFamilyFallback != null) {
    return _fontFamilyFallback.map((String family) => 'packages/$_package/$family').toList();
  }
  return _fontFamilyFallback;
}
```